diff --git a/lib/web/settings/components/extension_diff_live.ex b/lib/web/settings/components/extension_diff_live.ex
index 9a6b664..8496807 100644
--- a/lib/web/settings/components/extension_diff_live.ex
+++ b/lib/web/settings/components/extension_diff_live.ex
@@ -31,7 +31,7 @@ defmodule Bonfire.Me.Web.SettingsLive.ExtensionDiffLive do
   end
 
   def generate_diff(package, repo_path) do
-    case repo_diff(package, repo_path) do
+    case repo_latest_diff(package, repo_path) do
       {:ok, stream} ->
         path = render_diff(package, repo_path, stream)
 
@@ -68,7 +68,7 @@ defmodule Bonfire.Me.Web.SettingsLive.ExtensionDiffLive do
 
   end
 
-  def repo_diff(package, repo_path) do
+  def repo_latest_diff(package, repo_path) do
     path_diff = tmp_path("diff-#{package}-")
 
     with  :ok <- git_fetch(repo_path),
@@ -87,7 +87,7 @@ defmodule Bonfire.Me.Web.SettingsLive.ExtensionDiffLive do
       {:ok, stream}
     else
       error ->
-        Logger.error("Failed to create diff #{package} for #{repo_path} with: #{inspect(error)}")
+        Logger.error("Failed to create diff #{inspect package} for #{repo_path} at #{path_diff} with: #{inspect(error)}")
         :error
     end
   end
@@ -115,7 +115,7 @@ defmodule Bonfire.Me.Web.SettingsLive.ExtensionDiffLive do
            "diff.algorithm=histogram",
            "diff",
           #  "--no-index", # specify if we're diffing a repo or two paths
-           "--staged", # include staged changes
+          #  "--cached", # include staged changes (older git versions don't support the equivaled --staged)
            "--no-color",
            "--output=#{path_output}",
          ], repo_path) do
